packages/yuudachi/src/commands/moderation/case.ts

```diff
@@ -102,8 +91,10 @@ export default class extends Command<typeof CaseLookupCommand> {
 
 		if (cmd === 'history' && id) {
 			const data = await resolveMemberAndUser(interaction.guild, id);
+			const embed = await generateHistory(interaction, data, locale);
+
 			await interaction.editReply({
-				embeds: [truncateEmbed(await generateHistory(interaction, data, locale))],
+				embeds: [embed],
```

Comment:
Why aren't we truncating anymore?

Reply:
The function already truncates on the output, don't need to truncate again

Reply:
Maybe the function itself shouldn't truncate though.

Reply:
I will change it them. It might be a dumb question bc I really suck at code pattern, but isn't better to truncate in the function instead of repeatingon every instance?

Reply:
No, its a bit easier this way because this way we can, if we need to, pass untruncated content not to discord (if that case ever happens)

Reply:
Oh, got it, thankss for the explanation 😁

Reply:
Changed in 1daf11a